store/deposit-dates.js

```diff
@@ -34,25 +32,27 @@ class DepositDates extends Store {
     this.publicationDatesValidateUrl = `${baseUrl}/publication-dates-validate`
 
     this.retrieveDepositTimeLag()
-    this.loadDepositDatesCount()
     this.getPublicationDatesValidate()
     this.getCrossDepositLag()
   }
 
+  @computed
+  get totalCount() {
+    return this.timeLagData.reduce((acc, curr) => acc + curr.worksCount, 0)
+  }
+
+  @computed
+  get compliantCount() {
+    return this.timeLagData.reduce((acc, curr) => {
+      if (curr.depositTimeLag <= 90) return acc + curr.worksCount
+      return acc
+    }, 0)
+  }
+
   @computed
   get complianceLevel() {
-    const [total, compliant] = this.timeLagData.reduce(
-      (acc, curr) => {
-        acc[0] += curr.worksCount
-
-        if (curr.depositTimeLag <= 90) acc[1] += curr.worksCount
-        return acc
-      },
-      [0, 0]
-    )
-
-    if (total === 0) return 0
-    const level = (compliant / total) * 100
+    if (this.totalCount === 0) return 0
+    const level = (this.compliantCount / this.totalCount) * 100
```

Comment:
I am not sure this is better, a little hacky.

```suggestion
    const level = (this.compliantCount / (this.totalCount || 1)) * 100
```